### PR TITLE
Cell Efficiency, including Temperature Coefficient

### PR DESCRIPTION
**Pull request by ufshk**
Includes class for Solar Cells, and relevant attributes.

### CODE REVIEW

File: solar/cell.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        self.temperature = temperature
+        
+    def total_efficiency_decrease(self):
+        if self.temperature > 25:
+            efficiency = self.efficiency - self.temp_coefficient * (self.temperature - 25)
+            return efficiency
+        return self.efficiency
+    
+    def energy_converted(self)
+        return SolarDay.energy_received() * total_efficiency_decrease()
```
Comment (clarkevandenhoven) [inline solar/cell.py]: Should toss in a new line here for Python standards

File: solar/cell.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from solar import SolarDay
```
Comment (clarkevandenhoven) [inline solar/cell.py]: Comments at the top of the file explaining what it does are great for resuse

File: solar/cell.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+class SolarCell:
+    def __init__(self, area, temperature):
+        self.area = area
+        # efficiency of solar energy conversion to electrical energy
+        self.efficiency = .237 
+        # temp_coeff is the percentage decrease
+        # in the cells' efficiency for every degree above 25C
+        self.temp_coefficient = -.0029
```
Comment (clarkevandenhoven) [inline solar/cell.py]: Where does this number come from?

File: solar/cell.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        # the internal temperature of the cells/array
+        self.temperature = temperature
+        
+    def total_efficiency_decrease(self):
+        if self.temperature > 25:
```
Comment (clarkevandenhoven) [inline solar/cell.py]: What happens if the temperature is less than 25?

Comment (ufshk) [reply solar/cell.py]: Added comments to explain.

**Comment by clarkevandenhoven** [comment]
This is looking good. Please squash the commits into 1.

**Comment by ufshk** [comment]
I'm struggling to squash the commits, I think I did it correctly, but it now says I'm trying to merge 7 commits.

**Pull request merged**